### PR TITLE
fix: add more tsa url sanity check

### LINKED ISSUE #36: Add sanity check to TSA url when new http timestamper
We could add a sanity check on the url scheme and host to fail fast.

### PR DESCRIPTION
**Pull request by ghost**
This PR adds more tsa url sanity checks when new an http timestamper.

Resolves #36

### CODE REVIEW

**Review by JeyJeyGao — DISMISSED** [review]
LGTM

**Review by shizhMSFT — DISMISSED** [review]
LGTM

File: http.go [hunk 1]
```diff
@@ -53,9 +53,16 @@ func NewHTTPTimestamper(httpClient *http.Client, endpoint string) (Timestamper,
 	if httpClient == nil {
 		httpClient = &http.Client{Timeout: 5 * time.Second}
 	}
-	if _, err := url.Parse(endpoint); err != nil {
+	tsaURL, err := url.Parse(endpoint)
+	if err != nil {
 		return nil, err
 	}
+	if tsaURL.Scheme == "" {
```
Comment (priteshbandi) [inline http.go]: Should we validate http/https?

Comment (ghost) [reply http.go]: I'm seeing both http and https public tsa servers. (most them are actually http, for example, `http://timestamp.digicert.com`) @priteshbandi

Comment (ghost) [reply http.go]: Added check that requires the scheme to be either `http` or `https`.

**Review by priteshbandi — APPROVED** [review]
LGTM

**Review by JeyJeyGao — APPROVED** [review]
LGTM

**Pull request merged**